controller/src/main/java/cern/jarrace/controller/domain/Entrypoint.java

```diff
@@ -0,0 +1,54 @@
+package cern.jarrace.controller.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author timartin
+ * Domain class that represents a registered {@link Entrypoint}
+ */
+
+@Entity
+@Table(name = "ENTRYPOINT_TABLE")
+public class Entrypoint {
```

Comment:
this class has now been replaced in the new commit and it is immutable, I left it not immutable cause spring uses jackson to set the values, which I though were using setters, now I noticed that it is not true and I removed the set and left only the gets. I cannot force a final though cause jackson uses the default constructor and sets the parameters after